lm_eval/models/cm3.py

```diff
@@ -59,6 +61,10 @@ def __init__(self, truncate=False, api_key="dummy_key"):
         )[0]
         self.api_key = api_key
 
+    @property
```

Comment:
@lilisierrayu This is to avoid generating `<|/ file` tags, which the model can sometimes do when it thinks it's in a code doc. It's technically not backward compatible, for any current results that use greedy_until on CM3, but I doubt that it will cause issues.

Reply:
Do we need this for code application, or all other modalities?

Reply:
Just for code should be fine. I can put it into each code task instead.